lib/ex_backend/workflow/definition/francetv_acs.ex

```diff
@@ -19,37 +21,138 @@ defmodule ExBackend.Workflow.Definition.FrancetvAcs do
             enable: true,
             default: [mp4_path, ttml_url],
             value: [mp4_path, ttml_url]
+          },
+          %{
+            id: "source_hostname",
+            type: "credential",
+            default: "AKAMAI_REPLAY_HOSTNAME",
+            value: "AKAMAI_REPLAY_HOSTNAME"
+          },
+          %{
+            id: "source_username",
+            type: "credential",
+            default: "AKAMAI_REPLAY_USERNAME",
+            value: "AKAMAI_REPLAY_USERNAME"
+          },
+          %{
+            id: "source_password",
+            type: "credential",
+            default: "AKAMAI_REPLAY_PASSWORD",
+            value: "AKAMAI_REPLAY_PASSWORD"
+          },
+          %{
+            id: "source_prefix",
+            type: "credential",
+            default: "AKAMAI_REPLAY_PREFIX",
+            value: "AKAMAI_REPLAY_PREFIX"
+          },
+          %{
+            id: "source_ssl",
+            type: "credential",
+            default: "AKAMAI_REPLAY_SSL",
+            value: "AKAMAI_REPLAY_SSL"
           }
-        ] ++ get_akamai_source_parameters()
+        ]
       },
       %{
         id: 1,
         parent_ids: [0],
         required: [0],
-        name: "audio_extraction",
+        name: "job_ffmpeg",
+        label: "Extract audio",
+        icon: "queue_music",
         enable: true,
         parameters: [
           %{
             id: "input_filter",
             type: "filter",
             default: %{ends_with: [".mp4", ".isma"]},
             value: %{ends_with: [".mp4", ".isma"]}
+          },
+          %{
+            id: "command_template",
+            type: "string",
```

Comment:
Shouldn't it be a `template` instead?

Reply:
nope, it's the parameter passed to the worker.
Type template is not send to workers, it's just used to generate variable from a template in step_flow.